components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -2910,17 +2912,15 @@ private String base64Encode(String content) {
      * in the response.
      *
      * @param e         IdentityProviderManagementException
-     * @param errorEnum Error Message information.
      * @return APIError.
      */
-    private APIError handleIdPException(IdentityProviderManagementException e,
-                                        Constants.ErrorMessage errorEnum, String data) {
-
-        ErrorResponse errorResponse = getErrorBuilder(errorEnum, data).build(log, e, includeData(errorEnum, data));
+    private APIError handleIdPException(IdentityProviderManagementException e, String data) {
```

Comment:
won't there a case where `e.getErrorCode()` is null ? in that case, we need the `errorEnum`

Reply:
Addressed with [33231b3645c317e8cbbe7d2eaf7f6e7ef6e0f90a](33231b3645c317e8cbbe7d2eaf7f6e7ef6e0f90a)